pahfit/tests/test_classic_pack.py

```diff
@@ -30,7 +30,7 @@ def test_classic_pack():
     obs_y = obs_spectrum['flux'].to(u.Jy,
                                     equivalencies=u.spectral_density(obs_x))
 
-    pmodel = PAHFITBase(obs_x.value, obs_y.value, filename=packfilename, tformat='ipac')
+    pmodel = PAHFITBase(obs_x.value, obs_y.value, estimate_start=False, filename=packfilename, tformat='ipac')
```

Comment:
Since `estimate_start=False` is the default, you do not need to put include this in the PAHFITBase call.